Tasks/month.js

```diff
@@ -15,12 +16,14 @@ Months = [
   'dec',
 ];
 
-Month = (s) => {
-  l = Months.length;
-  for (let i = 0; i < l; i++) {
-    if (s.toLowerCase().startsWith(Months[i])) return i + 1;
+const monthNumber = month => {
+  const length = MONTHS.length;
```

Comment:
```suggestion
  const { length } = MONTHS;
```
Use object destructuring